src/App.php

```diff
@@ -290,12 +290,12 @@ protected function normalizeHeaders(array $headers): array
     {
         $res = [];
         foreach ($headers as $k => $v) {
-            if (is_numeric($k) && ($p = strpos($v, ':')) !== false) {
-                $k = substr($v, 0, $p);
-                $v = substr($v, $p + 1);
+            if (is_numeric($k) && ($p = mb_strpos($v, ':')) !== false) {
```

Comment:
see below, op on headers, I do not think this is good, this method should be also as fast as possible